### ISSUE TITLE
Post your screenshots!

### ISSUE THREAD

**Issue by rnkn** [rnkn/fountain-mode#114]
<img width="1317" alt="fountain-mode-1" src="https://user-images.githubusercontent.com/1256849/74585507-0a01e900-5029-11ea-9d89-b10595a103f0.png">

**Comment by rnkn** [comment]
On Linux Mint (with Emacs 24)

![linux-mint1](https://user-images.githubusercontent.com/1256849/74656282-8e936980-51d9-11ea-8c79-052cf3898f54.png)

![linux-mint2](https://user-images.githubusercontent.com/1256849/74656292-94894a80-51d9-11ea-88ec-ed369ffedb2d.png)

**Comment by barelief** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/186209/82069706-633af080-96dc-11ea-98bb-ea0085f9b3b3.jpg)

I must say fountain mode in emacs is jawdropping!!! I used atom fountain before. But now, I think, I need some amends...

PS. Is there a way to have outline view in a window on the right?

**Comment by rnkn** [comment]
Thanks. Glad you like it.

There are a couple of approaches depending on taste. There is [imenu-list](https://github.com/bmag/imenu-list) which will separately display section headings, scene headings, synopses, notes (each optional in `fountain-imenu-elements`). Or you could have a buffer with everything folded to `fountain-outline-custom-level` then call `fountain-outline-to-indirect-buffer (C-C C-x b)` to open the things you want in the main buffer.

**Comment by barelief** [comment]
> Thanks. Glad you like it.
> 
> There are a couple of approaches depending on taste. There is [imenu-list](https://github.com/bmag/imenu-list) which will separately display section headings, scene headings, synopses, notes (each optional in `fountain-imenu-elements`). Or you could have a buffer with everything folded to `fountain-outline-custom-level` then call `fountain-outline-to-indirect-buffer (C-C C-x b)` to open the things you want in the main buffer.

Thanks, William. I will post screenshot when I'll get it working. I've never used Emacs, so I'm excited to learn something new  :))

**Comment by barelief** [comment]
<img width="958" alt="Screenshot 2020-05-16 at 00 16 19" src="https://user-images.githubusercontent.com/186209/82096951-9a27fb00-970a-11ea-9a51-311233f9214d.png">

Voila! Emacs is tough, though...

**Comment by rnkn** [comment]
Emacs has a learning curve, but it is brief. Your best bet is to follow the tutorial with `C-h t` then read the FAQ (`C-h C-f`) and/or relevant parts of the manual (`C-h r`). Avoid Reddit and YouTube as that tends to be more people showing how clever they are. Always user the Customize interface (`M-x customize-group`) to set options because it has builtin error checking; only paste code into your init once you've hit the limitations of the builtin methods (but that will likely be years away).

That tutorial looks like it will install from GNU ELPA, which is a few years old and supposed to be removed. To add MELPA-stable: `M-x customize-option RET package-archives RET` then add
`https://stable.melpa.org/packages/` to the list. I had hoped MELPA would have good instructions for this, but they ask users to paste a bunch of confusing code into their init (just don't do it!).

**Comment by barelief** [comment]
...added Olivetti

![image](https://user-images.githubusercontent.com/186209/82114046-ee5ec980-9762-11ea-8789-cb9d796b88de.png)

**Comment by barelief** [comment]
*writeroom-mode* + *tao-yang theme*
update: I start to understand the power of `M-x` 

![Screenshot (16)](https://user-images.githubusercontent.com/186209/82119367-8a4dfc80-9786-11ea-816e-6c6da87ceba4.png)

**Comment by rnkn** [comment]
Nice theme!

**Comment by barelief** [comment]
pressing `S-TAB` w/ [Chocolate theme](https://github.com/SavchenkoValeriy/emacs-chocolate-theme) 

![chocolate_theme](https://user-images.githubusercontent.com/186209/82141853-2b998900-9841-11ea-9444-1e742701de1a.gif)

I have two questions: 
1. Is there a shortcut to open imenu-list with outline?
2. How do I display full synopses and notes in imenu (when I use `visual-line-mode` it still doesn't show the whole text)

**Comment by rnkn** [comment]
No.2 would be something to ask on the Imenu-list repo. It's an interface to Imenu, and Imenu is part of Emacs, so it's a couple of steps removed from here and I don't have it installed atm.

**Comment by kohled** [comment]
[nord-theme](https://melpa.org/#/nord-theme) + [writeroom-mode](https://melpa.org/#/writeroom-mode) + font(face) =  [iA Writer Duospace](https://github.com/iaolo/iA-Fonts/tree/master/iA%20Writer%20Duo)
<img width="1791" alt="fountain-nord-screenshot" src="https://user-images.githubusercontent.com/53094322/87260129-f0e95100-c47d-11ea-886e-7153b5ce312a.png">
Just wow. What a refined implementation of fountain while offering a LOT of functionality. Thank you for all your hard work.

Total emacs beginner. So I want to say for anyone else in such a state reading this, everything I needed to learn to install, customize, and use fountain-mode is contained in the README and in this thread. It's totally worth the brief learning curve.

@rnkn a question if I may-- My understanding thus far of `Do What I Mean (TAB)` is that it in SCENE HEADINGS, it auto-completes previously used text in the 'time' slot - e.g. DAY, NIGHT, etc. It also auto-completes character names used within the same scene. It does not auto-complete the 'location' slot in SCENE HEADINGS, nor does it auto-complete character names previously used in other scenes. Is this understanding correct? Is there some other function I've yet to discover?

Thank you again. Truly.

**Comment by rnkn** [comment]
@kohled thanks for the kind words, I'm glad you're getting some usefulness from it.

With regards to autocomplete, all of the things you mentioned are autocompleted, you just need to call `fountain-completion-update` in your screenplay (bound to `C-c C-x a` by default or menu-bar > Fountain > Update Auto-Completion). You can automatically do this when Fountain Mode starts by customizing `fountain-mode-hook` and checking the box for `fountain-completion-update`. This is off by default is because it can slow down old computers.

Then all character names are auto-completed, with the order as per the docstring in `fountain-completion-get-characters`:

> First, return second-last speaking character, then the last speaking character, followed by each previously speaking character within scene. After that, return characters from `fountain-completion-additional-characters` then `fountain--completion-characters` [this is all the rest of your characters, which gets updated with `fountain-completion-update` as per above.]

Within scene headings, locations are autocompleted after you type a scene heading prefix (set in `fountain-scene-heading-prefix-list`, defaults to "INT" "EXT" "EST" "INT./EXT." "INT/EXT" "I/E") after you have called `fountain-completion-update`. Then after the `fountain-scene-heading-suffix-sep` (which defaults to "-" or "--") the time of day is autocompleted based on `fountain-scene-heading-suffix-list` (which defaults to "DAY" "NIGHT" "CONTINUOUS" "LATER" "MOMENTS LATER"). Both of these are user options and so can be customized.

The easiest way to discover how things work is type `C-h k` and the type a key command or click a menu item, or type `C-h o` and type any command or option name, which will show a help window with clickable links to related commands and options.

I realise the completion documentation wasn't clear so I've linked `fountain-completion-at-point` to the `fountain-dwim` doc.

**Comment by kohled** [comment]
So awesome. Okay that is all very clear. Thanks for taking the time to clarify it. Big gratitude again. Nothing left to do now but get to writing!
